pyseir/inference/initial_conditions_fitter.py

```diff
@@ -54,20 +56,23 @@ def __init__(
         self.start_days_after_t0 = start_days_after_t0
         self.min_days_required = min_days_required
 
-        # Load case data
-        case_data = load_data.load_county_case_data()
-        self.cases = case_data[case_data["fips"] == fips]
+        timeseries = combined_datasets.get_timeseries_for_fips(fips)
+        latest_values = combined_datasets.get_us_latest_for_fips(fips)
+
+        self.cases = timeseries.data
+        # NOTE(chris): Should cases drop any empty rows? Thinking if there is a timeseries
```

Comment:
How about logging warning if the cases column is all nan?

Reply:
I wonder if it would be better to drop NaN rows.  The code below checks `if n_days < min_days_required` - maybe that n_days should represent non null days.